### PR TITLE
[doc-fix] - changed preference to settings for setting pycharm for pyflink ide setting section

### PR DESCRIPTION
**Pull request by adipolak**
## What is the purpose of the change

setting pycharm for pyflink project - fixing from going to reference (which dosn't exist) to settings

## Brief change log
- doc text update - changed preference to settings 

## Verifying this change

no test required for this

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: ( no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? ( no)

### CODE REVIEW

File: docs/content/docs/flinkDev/ide_setup.md [hunk 1]
```diff
@@ -260,7 +260,7 @@ versions.
 [Flake8](https://pypi.org/project/flake8/) is used to enforce some coding guidelines.
 
 1. Install flake8 for your Python interpreter using `pip install flake8`.
-2. In PyCharm go to "Preferences" → "Tools" → "External Tools".
+2. In PyCharm go to "Settings" → "Tools" → "External Tools".
```
Comment (Myasuka) [inline docs/content/docs/flinkDev/ide_setup.md:263]: Could you please also update this in Chinese documents like [line 172](https://github.com/apache/flink/blob/master/docs/content.zh/docs/flinkDev/ide_setup.md?plain=1#L172) and [line 210](https://github.com/apache/flink/blob/master/docs/content.zh/docs/flinkDev/ide_setup.md?plain=1#L210)?

**Review by Myasuka — COMMENTED** [review]
Thanks for creating this PR, please take a look at my comments.

File: docs/content/docs/flinkDev/ide_setup.md (hunk 1 above)
Comment (adipolak) [reply docs/content/docs/flinkDev/ide_setup.md:263]: hi, I am not fluent in Chinese, but I think line 172 is ok. fixing line 210 which discuss the Pycharm settings, incoming change.

**Review by Myasuka — APPROVED** [review]
LGTM

**Pull request closed without merging**